lib/please_cli/please_cli/decision_task.py

```diff
@@ -122,6 +122,7 @@ def get_deploy_task(index,
             'tools', 'deploy:HEROKU',
             project,
             '--heroku-app=' + deploy_options['heroku_app'],
+            '--heroku-dyno-type=' + deploy_options.get('heroku_dyno_type', 'web'),
```

Comment:
Sure. I was wondering what you would prefer here, and opted for the less intrusive one.